server.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
```

Comment:
was this server provided or did you figure it out as you progressed through the week? If you figured it out, Bravo! Good job!